packages/react-devtools-shared/src/devtools/views/Profiler/SidebarCommitInfo.css

```diff
@@ -45,7 +45,6 @@
 .DurationsList {
   list-style: none;
   margin: 0.25rem 0 0 0;
-  padding: 0;
```

Comment:
this does not make any sense here

Reply:
Yes it does. Lists have padding by default. This disables that.